charts/opentelemetry-demo/examples/public-hosted-ingress/rendered/component.yaml

```diff
@@ -1857,29 +1857,3 @@ spec:
           resources:
             limits:
               memory: 20Mi
----
-# Source: opentelemetry-demo/templates/component.yaml
-apiVersion: "networking.k8s.io/v1"
```

Comment:
@puckpuck I believe the new service object refactor broke the `otel-demo.ingress` template.

Reply:
fixed